### ISSUE TITLE
SaveAs after server finishes starting writes wrong path in new location and can delete measures from old location

### ISSUE THREAD

**Issue by DavidGoldwasser** [NREL/OpenStudio#2702]
I made a new issue for this vs. #2697 so comments and documentation related to them don't get mixed up. In this case I followed the best possible path, allowing the server to fully start on my project before initiating a SaveAs operation. 

This commit represents the SaveAs operation
https://github.com/DavidGoldwasser/PAT_projects/commit/0a68d8ae5c2a51e53bda775345c4a8790227bb49

The commit is on a new branch just for this issue
https://github.com/DavidGoldwasser/PAT_projects/tree/save_as_after_server_starts

Steps carried out between this commit and the last commit.

1. Launch PAT and Opened "PAT_model_articulation_test_a"
2. Wait for server to fully start
3. Choose SaveAs and picked "temp" for name 
4. Removed "tariff_selection_flat" measure from workflow in "temp" PAT project in GUI
5. Quit project (I didn't have to save, that was done when I removed measure)
6. Committed projects to repository

Below is summary of what I observed. 
1. After step 3 there were no changes to "PAT_model_articulation_test_a" as shown in the screenshot below
2. After step 3 the new "temp" project was made, but the "projectName" for it still said "PAT-model_articulation_test_a" and all of the file paths still point to the old project.
3. After step 3 despite what item 2 above shows about projectName in pat.json file, the GUI shows "temp" as the project name.

(Images below relate to items 1-3)

![screen shot 2017-07-26 at 9 17 40 am](https://user-images.githubusercontent.com/4857055/28629978-3b5e953a-71e6-11e7-9548-cfa65d79cad2.png)

![screen shot 2017-07-26 at 9 18 19 am](https://user-images.githubusercontent.com/4857055/28630008-49b86dea-71e6-11e7-82b8-8d0cbef5d5f8.png)

4. I wanted to delete a measure from the workflow to see where it was deleted from, and unfortunately it does delete the measure files from the wrong project. I would have to assume that updating measure would have the same outcome. See image below for change log after step 4 of removing the tariff measure.

![screen shot 2017-07-26 at 9 21 51 am](https://user-images.githubusercontent.com/4857055/28630190-d4abafd4-71e6-11e7-9086-b96a340a4c58.png)

**Comment by DavidGoldwasser** [comment]
5. Also after step 4, the pat.json in  "PAT_model_articulation_test_a" that was originally not altered after the save as, now is named "temp" and the tariff measure was removed from the original project and new the new project.
https://github.com/DavidGoldwasser/PAT_projects/commit/0a68d8ae5c2a51e53bda775345c4a8790227bb49#diff-3ec654d733743e30f4ad26c34a47204eL2346

![screen shot 2017-07-26 at 10 07 49 am](https://user-images.githubusercontent.com/4857055/28631449-7f965c2a-71ea-11e7-87ce-a183408a652d.png)

**Comment by DavidGoldwasser** [comment]
After quitting PAT (after a SaveAs) and then re-launching PAT and loading the projects, the paths seem to correct to be local to the project folder, it seems like that not happening right at the save ias is where a lot of issues are coming from, although that doesn't explain why the wrong pat.json is still being edited after Save as.

![screen shot 2017-07-26 at 10 19 03 am](https://user-images.githubusercontent.com/4857055/28631983-46adb032-71ec-11e7-9fe0-3e7eecad1781.png)
